index.html

```diff
@@ -49,5 +49,187 @@ <h1 id="name">
+                    <li>2015</li>
+                </ul>
+                <p class="work-details">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="work-tools">
+                    <li>HTML</li>
+                    <li>CSS</li>
+                    <li>Javascript</li>
+                </ul>
+                <button class="btn">See Project</button>
+            </div>
+        </div>
+    </section>
+
+    <section id="about">
+        <div class="about-container">
+            <p class="myself">About Myself</p>
+            <p class="about-details">
+                Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+            </p>
+            <h4 class="connect">let's connect</h4>
+            <!-- <a href="#" class="connect">let's connect</a> -->
+            <ul class="socials">
+                <li><a href="" >
+                    <i><img class="icons" src="/assets/images/icons/github.svg" alt=""></i>
+                </a></li>
+                <li><a href="" >
+                    <i><img class="icons" src="/assets/images/icons/Linkedin.svg" alt=""></i>
+                </a></li>
+                <li><a href="" >
+                    <i><img class="icons" src="/assets/images/icons/peace.svg" alt=""></i>
+                </a></li>
+                <li><a href="" >
+                    <i><img class="icons" src="/assets/images/icons/twitter.svg" alt=""></i>
+                </a></li>
+                <li><a href="" >
+                    <i><img class="icons" src="/assets/images/icons/microverse.svg" alt=""></i>
+                </a></li>
+            </ul>
+            <button class="btn resume">Get my resume</button>
+        </div>
+        <ul class="skills">
+            <li class="languages">
+                <div class="languages skill-group">
+                    <p class="title">languages</p>
+                    <div class="union-icon">
+                        <img class="active" src="/assets/images/icons/Union.svg" alt="">
+                    </div>
+                </div>
+                <ul class="language-list">
+                    <li>
+                        <p><img src="/assets//images//skillset/languages/javascript.svg" alt=""></p>
+                        <p class="lang-name">JavaScript</p>
+                    </li>
+                    <li>
+                        <p><img src="/assets//images//skillset/languages/html.svg" alt=""></p>
+                        <p class="lang-name">HTML</p>
+                    </li>
+                    <li>
+                        <p><img src="/assets//images//skillset/languages/css.svg" alt=""></p>
+                        <p class="lang-name">CSS</p>
+                    </li>
+                </ul>
```

Comment:
Awesome job implementing the previous reviewer's suggestion to use an `unordered list` to display your languages list. I just have a couple of friendly and optional suggestions to make it even more consistent with the Figma template:

### Figma design:
![image](https://user-images.githubusercontent.com/107952724/234999600-d8ac2277-64e3-45bc-ac55-bf3a1e91c190.png)

### Your design:
![image](https://user-images.githubusercontent.com/107952724/234998576-96ca3517-3913-4c06-82c8-1ddc70981bc3.png)
----
- [ ] **[OPTIONAL]** What do you think if we add a bit more of `gap` between the icon and the name of your language? 🎯
- [ ] **[OPTIONAL]** It might be worth checking the left and right **padding** of your section. Currently, you're using just `16px`, while the template uses` 24px` for the parent and a bit more for the children. This would help to ensure consistency throughout your portfolio. 🎯